### DISCUSSION TITLE
Welcome to node-red-contrib-python-venv Discussions!

### DISCUSSION THREAD

**Discussion by 404background** [404background/node-red-contrib-python-venv#44] [Announcements]
## 👋 Welcome!
Welcome to the Discussions page!

This page is for sharing flows that use the python-venv node.
I’m the developer, and I’d love to see how you are using it!

Of course, if sharing your flows is difficult, you are welcome to share examples or ideas instead!
Feel free to ask questions or share your ideas.

**Comment by shobita02** [comment]
Hii im trying to use the python-venv node in one of my flows, and every now and then the flow fails due to an error from that node.
However i cant seem to get to the root of the error because i can't log the entire error message in debug and the error isn't being caught by a catch node.

is there any way i can extract the full error message from the node?
